app/src/main/java/com/concordium/wallet/onboarding/ui/di/ModuleLoader.kt

```diff
@@ -0,0 +1,17 @@
+package com.concordium.wallet.onboarding.ui.di
+
+import org.koin.core.context.loadKoinModules
+import org.koin.core.context.unloadKoinModules
+import org.koin.core.module.Module
+
+abstract class ModuleLoader {
+    abstract val modules: List<Module>
+
+    fun load() {
+        loadKoinModules(modules)
+    }
+
+    fun unload() {
+        unloadKoinModules(modules)
+    }
```

Comment:
According to Studio, none of these methods are used.

Reply:
we can keep it this way for now, I'll change it when necessary